test/conformance/CMakeLists.txt

```diff
@@ -12,7 +13,7 @@ function(add_test_adapter name adapter)
     add_test(NAME ${TEST_NAME}
         COMMAND ${CMAKE_COMMAND}
         -D TEST_FILE=${Python3_EXECUTABLE}
-        -D TEST_ARGS="${UR_CONFORMANCE_TEST_DIR}/cts_exe.py --test_command ${CMAKE_BINARY_DIR}/bin/${TEST_TARGET_NAME}"
+        -D TEST_ARGS="${UR_CONFORMANCE_TEST_DIR}/cts_exe.py --test_command ${CMAKE_BINARY_DIR}/bin/${TEST_TARGET_NAME} --NUMBER_OF_DEVICES=${NUMBER_OF_DEVICES}"
```

Comment:
`NUMBER_OF_DEVICES` as an input argument should be lowercase, similar to `test_command`.

Reply:
Done